engineering_tools/carma_script_extensions/cloud-sim

```diff
@@ -158,104 +158,164 @@ cloud-sim__start() {
     
     # Call cloud formation
     local STACK_NAME="CloudSimStack"$(echo ${USER_ARN} |  grep -oh "/.*@" | sed "s|@|-|g" | sed "s|\.|-|g" | sed "s|[^A-Za-z0-9]|-|g")
-
```

Comment:
Just to simplify this a little, none of the logic in red here has been removed. These calls have moved to a different line.
In the green, Lines 214-314 have just been indented - no change in code